src/Symfony/Component/Form/Tests/ChoiceList/Loader/AbstractChoiceLoaderTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\ChoiceList\Loader;
+
+use Symfony\Component\Form\ChoiceList\LazyChoiceList;
+
+/**
+ * @author Jules Pietri <jules@heahprod.com>
+ */
+abstract class AbstractChoiceLoaderTest extends \PHPUnit_Framework_TestCase
```

Comment:
@HeahDude We can still extract common logic in an abstract base class then if needed. I would remove this class for now too.